### DISCUSSION TITLE
sharing across RStudio projects / gh repos

### DISCUSSION THREAD

**Discussion by yonicd** [ropensci/targets#757] [General]
I saw in the docs that you explain how to share across targets projects, is it possible to extend that across repos via the targets yaml?

for example i have 

gh_org/program_AAA_trial_1
gh_org/program_AAA_trial_2

and they are in the same trial program (AAA), and share common data setup functionality. 

I would want to preprocess under one pipeline and then distribute some outputs to trial_1 specific analysis in the trial_1 repo and then some trial specific analysis in trial_2 repo.

**Comment by wlandau** [comment]
https://books.ropensci.org/targets/projects.html#sharing-targets assumes both projects share a local file system. I consider GitHub integration out of scope because data files are usually too big to upload to a remote repo.
